proto/cosmos/group/module/v1/module.proto

```diff
@@ -10,7 +10,7 @@ import "amino/amino.proto";
 // Module is the config object of the group module.
 message Module {
   option (cosmos.app.v1alpha1.module) = {
-    go_import: "github.com/cosmos/cosmos-sdk/x/group"
+    go_import: "cosmossdk.io/x/group"
```

Comment:
Can we get a PR on cosmos/vanity?

Reply:
Sure, let me start off with that,

Reply:
@julienrbrt I've mailed out https://github.com/cosmos/vanity/pull/51